packages/sign_in_with_apple/lib/src/widgets/sign_in_with_apple_button.dart

```diff
@@ -2,15 +2,18 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:sign_in_with_apple/src/widgets/apple_logo_painter.dart';
 
-/// Style according to
+const _appleIconSize = 28.0;
```

Comment:
This should scale with the height of the button right? So should be use a factor (like `28/44`)?

Reply:
Yeah, this might also improve the aligning of the button with a custom height